kernel/arm64/dgemv_kernel_sve.c

```diff
@@ -0,0 +1,41 @@
+#include "common.h"
```

Comment:
I don't think this file is used and is already implemented in `kernel/arm64/gemv_n_sve.c` / `kernel/arm64/gemv_t_sve.c`